src/app/components/MediaLoader/index.tsx

```diff
@@ -105,6 +196,7 @@ const MediaLoader = ({ blocks, className }: Props) => {
       css={styles.figure}
       className={className}
     >
+      {showAds && <AdvertTagLoader queryString={queryString} />}
```

Comment:
Could we use the `useLocation` hook directly inside the `AdvertTagLoader` component rather than passing `queryString` down as a prop? May just make it more portable.